### PR TITLE
[WIP] add webpack & react stuff & a view for sulten index

### PR DESCRIPTION
**Pull request by alfiehub**
a bit big commit, but w/e

### CODE REVIEW

File: react/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "babel-preset-react": "^6.24.1",
+    "webpack": "^3.5.5"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack --config webpack.config.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC"
```
Comment (aleksanb) [inline react/package.json:28]: Any thoughts on whether this should be MIT like the main repo?
I see that  ISC is compatible with MIT though.

File: react/sulten/index.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import Root from './views/Root'
+
+console.log("Hello from react!")
```
Comment (aleksanb) [inline react/sulten/index.js]: log statement

**Review by aleksanb — APPROVED** [review]
LGTM. Is the PR still WIP?

File: react/package.json (hunk 1 above)
Comment (alfiehub) [reply react/package.json:28]: Sure, I guess we could just set the license to MIT instead. ISC was just the default value, lol 😂

**Comment by tollefj** [comment]
Changes moved to a separate branch with another repo for react files. Closed.

**Pull request closed without merging**